Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Great !</p><p>Can you tell me - why the first 2 seasons include much more goals ?</p><p> </p><p>Find a SIMPLE way to fix that in the graph (i.e., "normalize" the goals count).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Of course, this is because in the first two seasons there were 2 more teams, meaning also more games that are an opportunity to score goals.</p>

Reply:
<p>Good</p>